### PR TITLE
Complete more_methods.md for week_two_code_challenges

### PR DESCRIPTION
**Pull request by BrianZCheung**
This PR completes more methods for week_two_code_challenges

### CODE REVIEW

File: brian_cheung/week_two_challenges/more_methods.md [hunk 1]
```diff
@@ -36,13 +44,35 @@ When `fizz_buzz(max_number)` is executed, it should do the following for every n
...
 
 ## Manipulative Methods
 
 1. Declare a method called `big_backwards` that takes a single string as an argument and returns that string backwards and uppercase.
 
 ```ruby
-# write your code here
+def big_backwards(word)
+  puts "#{word.upcase.reverse}"
+end
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/more_methods.md]: No need to interpolate here; the word is already a string.

File: brian_cheung/week_two_challenges/more_methods.md [hunk 2]
```diff
@@ -52,7 +82,9 @@ Example output:
 "The word 'dog' has 3 letters!"
 ```
 ```ruby
-# write your code here
+def informed_counter (word)
+  puts word.split('').count
+end
 ```
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/more_methods.md]: Don't forget to read the directions fully: the output should return the string, "The word {word} has {number} letters!"

File: brian_cheung/week_two_challenges/more_methods.md [hunk 3]
```diff
@@ -64,11 +96,18 @@ big_ordered_array(test_array)
 # output => ['I', 'OF', 'FULL', 'LOVE', 'ARRAYS', 'MAKING', 'STRINGS']
 ```
 ```ruby
-# write your code here
+def big_ordered_array (array)
+  puts array.map {|word| word.upcase }.sort_by { |word| word.length }.group_by(&:length).map{ |itterate, array| array.sort}.flatten
+end
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/more_methods.md]: What's this all about? `.map{ |itterate, array| array.sort}`

File: brian_cheung/week_two_challenges/more_methods.md [hunk 4]
```diff
@@ -64,11 +96,18 @@ big_ordered_array(test_array)
...
 
 ```ruby
-# write your code here
+def lucky_number (number)
+  if number % 13
+    puts "Unlucky!"
+  else
+    puts "Lucky!"
+end
 ```
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/more_methods.md]: Read those directions again 😉

File: brian_cheung/week_two_challenges/more_methods.md (hunk 3 above)
Comment (BrianZCheung) [reply brian_cheung/week_two_challenges/more_methods.md]: iterate goes to each hash and the array passes in the hash to be sorted alphabetically with .sort then flatten merges them all into one array

**Comment by gretchenziegler** [comment]
Uh, looks like that "minor fixes" commit undid all of the changes I asked you to make 😕 And I don't think we want to add those untracked files, either. You should reset the last two commits (`git reset --hard HEAD^^`) and then force push to remove these last two commits from the PR.

File: brian_cheung/week_two_challenges/more_methods.md [hunk 5]
```diff
@@ -64,11 +93,24 @@ big_ordered_array(test_array)
...
 4. Declare a method called `lucky_number` that takes two numbers as arguments and returns "Unlucky!" if the sum of those numbers is divisible by 13, and "Lucky!" if not.
 
 ```ruby
-# write your code here
+def lucky_number (number1, number2)
+  if (number1 + number2) % 13 == 0
+    puts "Unlucky!"
+  else
+    puts "Lucky!"
+end
```
Comment (gretchenziegler) [inline brian_cheung/week_two_challenges/more_methods.md]: One last thing: you're missing something here that will cause this code to break...

Comment (BrianZCheung) [reply brian_cheung/week_two_challenges/more_methods.md]: ...crap.. whoops...

**Review by gretchenziegler — APPROVED** [review]

**Pull request merged**